README_py.md

```
@@ -37,62 +36,51 @@ PYBIND11_MODULE(EmotiBitPacket, m) {
   - Build the files created in the previous step to create the `.pyd` file
 
 ## Requirements
-### Creating python-env with pybind11
-#### Using venv
+### Setting up Python Virtual Environment
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)